### ISSUE TITLE
Error when trying to create a new custom field

### ISSUE THREAD

**Issue by bartonski** [bywatersolutions/rtenhancements#24]
Got the following error message

`Could not create CustomField: Internal Error: Couldn't execute the query 'INSERT INTO CustomFields (LastUpdated, MaxValues, Creator, LookupType, Created, Type, UniqueValues, Description, Pattern, ValuesClass, CanonicalizeClass, Disabled, Name, LastUpdatedBy, BasedOn, RenderType) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)'Unknown column 'UniqueValues' in 'field list'`

**Comment by bartonski** [comment]
this is in `/Admin/CustomFields/Modify.html`

**Comment by margaret-bywater** [comment]
This isn't a problem on current test sites. So closing this issue.

**Comment by sbbeachvball** [comment]
I ran into this problem on rt-5.0.1.  We have been running RT for quite some time, but have not attempted using Custom Fields till now.  I expect that there was a bug in one of the upgrade-database scripts somewhere in our RT updates.  The two fields I have seen missing are UniqueValues and CanonicalizeClass.  I would love to hear if there is a way to clean this up and get us more functional without having to start from scratch.

In a followup on this.  I checked and it seems like only the updates from 4.4.2 were missed.  I checked other tables and as far as I could tell, all other sql updates before and after had been applied to the db.  So not sure why that one was missed.  I manually ran the 2 sql commands from that script to get my table current.  I could then create Custom Fields as expected.
